Libraries/Components/AppleTV/tagForComponentOrHandle.js

```diff
@@ -0,0 +1,34 @@
+// Replacement for findNodeHandle, since we need this for converting
+// components to tags when rendering nextFocus props
+// @flow
+
+// TODO: make this work for Fabric
+
+// import {findNodeHandle} from '../../Renderer/shims/ReactNative';
```

Comment:
What's the reason for not use the original? Seems to do the same thing and some more.
https://github.com/facebook/react-native/blob/41cbccd98dd6c98d1f662674164cf455105a1359/Libraries/Renderer/implementations/ReactNativeRenderer-prod.js#L8067